src/components/smart/SystemFields.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <ul data-cy="systemfield-ul">
+    <li data-cy="systemfield-status"><span>Status:</span> {{ props.status || '-' }}</li>
+    <li data-cy="systemfield-aangemaakt-door">
+      <span>Aangemaakt door:</span> {{ props.createdBy || '-' }} op {{ createdAt }}
+    </li>
+    <li data-cy="systemfield-laatst-bewerkt" v-if="props.updatedBy">
+      <span>Laatst bewerkt door:</span> {{ props.updatedBy }} op {{ updatedAt }}
+    </li>
+    <slot data-cy="systemfield-slot"></slot>
+  </ul>
+</template>
+
+<script setup lang="ts">
+import type { ISystemFieldsProps } from '@models/system-fields';
+import { computed } from 'vue';
+
+const props = defineProps<ISystemFieldsProps>();
+const createdAt = computed(() => (props.createdAt ? formatDate(props.createdAt) : '-'));
+const updatedAt = computed(() => (props.updatedAt ? formatDate(props.updatedAt) : '-'));
+
+const formatDate = (dateString: string) => {
+  try {
+    const date = new Date(dateString);
+    const options1: Intl.DateTimeFormatOptions = {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit',
+    };
```

Comment:
Vermoed dat er ook nog iets van timezone config nodig is aangezien de Cypress testen in github falen met 2u verschil :)